helm-chart/renku/templates/secrets.yaml

```diff
@@ -37,3 +37,26 @@ type: Opaque
 data:
   {{ .Values.redis.existingSecretPasswordKey }}: {{ .Values.redis.password | b64enc | quote }}
 {{- end }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: "{{ .Release.Name }}-rabbitmq-admin"
+  labels:
+    app: {{ template "renku.name" . }}
+    chart: {{ template "renku.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+stringData:
+  config.yaml: |
+    admin:
+      {{- if .Values.global.rabbitmq.enabled }}
+      username: {{ required ".Values.rabbitmq.auth.username must be defined" .Values.rabbitmq.auth.username | quote }}
+      password: {{ required "Fill in .Values.rabbitmq.auth.password with `openssl rand -hex 32`" .Values.rabbitmq.auth.password | quote }}
+      {{- else }}
+      username: {{ required ".Values.global.rabbitmq.external.admin.username must be defined" .Values.global.rabbitmq.external.admin.username | quote }}
+      password: {{ required "Fill in .Values.global.rabbitmq.external.admin.password with `openssl rand -hex 32`" .Values.global.rabbitmq.external.admin.password | quote }}
+      {{- end }}
```

Comment:
I think it'd be good if we generated/reused passwords the same way I have in #2122 from the start. Especially if we're allowing for an external rabbitmq to be used.